scripts/protocgen.sh

```diff
@@ -25,6 +25,11 @@ done
 
 cd ..
 
+# temporary import hack to use cosmos-sdk implementation of Any type.
+# check https://github.com/celestiaorg/celestia-app/issues/507 for more information.
+sed -i 's/types "github.com\/celestiaorg\/celestia-app\/codec\/types"/types "github.com\/cosmos\/cosmos-sdk\/codec\/types"/g' \
+ github.com/celestiaorg/celestia-app/x/qgb/types/query.pb.go
```

Comment:
agree with @rootulp that we might need to reopen or create a new issue to make sure we don't forget about this

Reply:
https://github.com/celestiaorg/celestia-app/issues/744